lms/envs/devstack_docker.py

```diff
@@ -25,12 +25,6 @@
 CREDENTIALS_INTERNAL_SERVICE_URL = 'http://edx.devstack.credentials:18150'
 CREDENTIALS_PUBLIC_SERVICE_URL = 'http://localhost:18150'
 
-OAUTH_OIDC_ISSUER = '{}/oauth2'.format(LMS_ROOT_URL)
```

Comment:
Do we really no longer use the issuer, or does this need to be called something like `OAUTH_ISSUER`?

Reply:
it does not seem to be called anywhere

Reply:
I this you are correct and it should be called something else in some places

Reply:
especially when it is used to set JWT_ISSUER